PWGUD/AQC/udQCmidRap.cxx

```diff
@@ -459,8 +459,8 @@ struct UDQCmid {
 
   //.....................................................................................................................
   // Distribution of number of PV contributors for all collisions and those with empty FT0
-  void processFewProng(CC const& collision, BCs const& /*bct0s*/,
-                       aod::FT0s const& /*ft0s*/, aod::FV0As const& /*fv0as*/, aod::FDDs const& /*fdds*/)
+  void processFewProng(CC const& collision, BCs const& bct0s,
+                       aod::FT0s const& ft0s, aod::FV0As const& fv0as, aod::FDDs const& fdds)
```

Comment:
Please don't revert fixes.

Reply:
There are some issues with this task, unresolved commits from the past creating many conflicts in the master branch which is why this PR is done separately. Working on fixing the conflict in the main branch. Thanks for your comments, will do a new PR with fixes.